numpy/core/defchararray.py

```diff
@@ -977,6 +983,15 @@ def join(sep, seq):
     See Also
     --------
     str.join
+
+    Examples
+    --------
+    >>> np.char.join('-','osd')
+    array('o-s-d', dtype='<U5')
+
+    >>> np.char.join(['-','.'],['ghc','osd'])
+    array(['g-h-c', 'o.s.d'], dtype='<U5')
```

Comment:
Extremely nitpicky comment, please follow PEP8 for the examples: 

```suggestion
    >>> np.char.join('-', 'osd')
    array('o-s-d', dtype='<U5')

    >>> np.char.join(['-', '.'], ['ghc', 'osd'])
    array(['g-h-c', 'o.s.d'], dtype='<U5')
```